docs/admin/_kubeadm/kubeadm_alpha_phase_addon_all.md

```diff
@@ -0,0 +1,25 @@
+
+Install all addons to a Kubernetes cluster
+
+### Synopsis
+
+
+Install all addons to a Kubernetes cluster
+
+```
+kubeadm alpha phase addon all
+```
+
+### Options
+
+```
```

Comment:
Is this stuff generated?  I'd be a fan to reduce maintenance by linking in auto-generated stuffs.

Reply:
yes, all the `synopsis`, `options`, `inherited options` paragraphs are generated starting from kubeadm code.
Generated files are then copied in the website and, using an inline jekyll command, included into the pages you see in the menu, where there is also some additions/extra content not derived from cobra.commands.
For alpha commands, there are no additional context so far (only page structure)

Reply:
>  For alpha commands, there are no additional context so far (only page structure)

We should add that eventually, but please in an other PR